primer.md

```diff
@@ -677,7 +682,9 @@ came from and where it might be going. This prevents tooling to facilitate
 successful event delivery and consumers from knowing what to do with event data.
 
 CloudEvents offers useful metadata which middleware and consumers can rely upon
-to facilitate event routing, logging, delivery and receipt.
+to facilitate event routing, logging, delivery and receipt. (Not sure if we 
```

Comment:
You should remove the commentary since we can't merge with it in there.
It's not meant to "impact" it, but it is meant to help "facilitate" it.

Reply:
makes sense.